app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -261,7 +261,9 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
         product = BulkProducts.find product.id
         variant.unit_value  = parseFloat(match[1].replace(",", "."))
         variant.unit_value  = null if isNaN(variant.unit_value)
-        variant.unit_value *= product.variant_unit_scale if variant.unit_value && product.variant_unit_scale
+        if variant.unit_value && product.variant_unit_scale
+          variant_unit_value = variant.unit_value * product.variant_unit_scale
+          variant.unit_value = parseFloat(variant_unit_value.toFixed(8))
```

Comment:
I think I'd prefer use `js-big-decimal` (https://www.npmjs.com/package/js-big-decimal)
something like:
```
bigDecimal.multiply(variant.unit_value, product.variant_unit_scale, 2)
```

Reply:
hii @jibees 
I have used the package `js-big-decimal` but the specs have started to fail because specs are unable to find the method `bigDecimal.multiply`. Is it because we have included `js-big-decimal` as Dependenciesand not devDependencies?

Reply:
I guess that `bigDecimal.multiply` should be imported or mocked for the specs environment ...

Reply:
(not that easy actually)

Reply:
hii @jibees 
Thanks for the above suggestions. I have made the changes in the specs. Can you please re review it.
Thanks